.travis.yml

```diff
@@ -1,18 +1,30 @@
 language: scala
-scala:
-- 2.10.6
-jdk:
-- oraclejdk8
+
+matrix:
+  include:
+    - scala: 2.12.3
+      jdk: oraclejdk8
+      env:
+        - SBT_VERSION=1.0.1
+# TODO test sbt 0.13.16/(2.12.3) vs 0.13.16(2.10.6) in matrix
+    - scala: 2.12.3
+      jdk: oraclejdk8
+      env:
+        - SBT_VERSION=0.13.16
+
 before_install:
-- openssl aes-256-cbc -K $encrypted_5cd1f76c4e65_key -iv $encrypted_5cd1f76c4e65_iv
-  -in secring.gpg.enc -out secring.gpg -d
-- export PATH=${PATH}:./vendor/bundle
+  - openssl aes-256-cbc -K $encrypted_5cd1f76c4e65_key -iv $encrypted_5cd1f76c4e65_iv
+    -in secring.gpg.enc -out secring.gpg -d
+  - export PATH=${PATH}:./vendor/bundle
+
 install:
-- rvm use 2.2.3 --install --fuzzy
-- gem update --system
-- gem install sass
-- gem install jekyll -v 3.2.1
+  - rvm use 2.2.3 --install --fuzzy
+  - gem update --system
+  - gem install sass
+  - gem install jekyll -v 3.2.1
+
 script:
-- sbt orgScriptCI
+  - sbt ^^$SBT_VERSION orgScriptCI
+
 after_success:
-- sbt orgAfterCISuccess
+  - sbt ^^$SBT_VERSION orgAfterCISuccess
```

Comment:
Thinking more about the eventual `publishMicrosite`, it should only happen one of the sbt builds.
And also bumping `version.sbt`/whatever commits travis does after a merge to master.

Reply:
Yes, I agree. We could use a similar approach like the one we've used with the different Scala versions:
https://github.com/47deg/sbt-org-policies/blob/master/core/src/main/scala/sbtorgpolicies/runnable/model.scala#L25